### DISCUSSION TITLE
TuringCourses/math_phys/math_analysis2/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#57] [General]
# TuringCourses/math_phys/math_analysis2/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/math_phys/math_analysis2/

**Comment by MTDickens** [comment]
级数“（闭区间内）可逐项求导”充分条件是：
1. 原级数点态收敛
2. 级数的逐项求导形式一致收敛
3. 逐项求导的每一项都连续

19级的 **“ 已 经 证 得 该 三 角 级 数 一 致 收 敛 ， 则 对 其 求 导 可 逐 项 求 导”** 这句话意思的是：”我们已经证明了三角级数一致收敛（因此，该三角级数也点态收敛）。下面我们只要再证明逐项求导的级数也是一致收敛的（而且每一项都连续，不过这是显而易见的），就可以证明：级数的导数等于导数的级数，而且连续“

**Reply by ZhouTimeMachine** [reply]
感谢为我解释，检查了我当时写解答的时候参考的定理，这里应该是我写得不好，建议你们考完之后 remake 一份好的解答

**Comment by Fairicle** [comment]
数分二相对于数分一，对证明的要求变低了一些，考试的重点也在计算和定理的运用上。如果单从考试的角度来说，cjh 老师的风格（有许多例题，解题过程很清晰）非常适配。不过本人是 rhj 老师的忠实粉丝(x)，火老师在数分二也会把大部分证明完整推一遍，让你明白这些公式是怎么来的，从而能让你对这些知识有更好的掌握和理解，而不是只是背公式和硬套。

和数分一一样，我推荐大家以书本为主，习题如果能做完应该就很厉害了。不一定对每个证明都掌握，但起码要知道它们的条件、结论，掌握少部分比较重要的定理的证明。资料和历年卷很多，但是抛开书本只用教辅的学习方式在这门课并不推荐，所谓基础不牢地动山摇（笑）。
